modules/serverless-typescript-context-object-reference.adoc

```diff
@@ -0,0 +1,130 @@
+[id="serverless-typescript-context-object-reference_{context}"]
+= TypeScript context object reference
+
+The `context` object has several properties that can be accessed by the function developer.
+
+[id="serverless-typescript-context-object-reference-log_{context}"]
+== log
+
+Provides a logging object that can be used to write output to the cluster logs. The log adheres to the link:https://getpino.io/#/docs/api[Pino logging API].
+
+.Example log
+[source,javascript]
+----
+Function handle(context) {
```

Comment:
```suggestion
function handle(context: Context) Record<string,string> {
```

Reply:
Gah, my bad, I've used the wrong function header in some examples :/